packages/expo-notifications/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### 🐛 Bug fixes
 
+- [ios] add `NotificationEmitter` as a `NotificationCenterManager` delegate inside `onCreate` ([#36930](https://github.com/expo/expo/pull/36930) by [@jmysliv](https://github.com/jmysliv))
```

Comment:
let's make it easier to understand what bug this fixes, focus on the bugfix rather than the technicals pls.